firefox-ios/Client/Frontend/Theme/ThemedTableViewController.swift

```diff
@@ -77,6 +77,7 @@ class ThemedTableViewController: UITableViewController, Themeable, InjectedTheme
         super.viewDidLoad()
         tableView.register(cellType: ThemedTableViewCell.self)
         tableView.register(cellType: ThemedCenteredTableViewCell.self)
+        tableView.register(cellType: ThemedLearnMoreTableViewCell.self)
```

Comment:
I think we should only register the cell in the view controllers it is actually used in.